### ISSUE TITLE
add support for game assets from Wipeout 2097

### ISSUE THREAD

**Issue by SingingBush** [phoboslab/wipeout-rewrite#14]
Just read the article about the model viewer and saw that https://github.com/phoboslab/wipeout supports Wipeout 2097 tracks. Are there any plans to either do a Wipeout 2097 rewrite or for this codebase to also support track data from Wipeout 2097?

**Comment by tigrouind** [comment]
Wipeout 64 also used same track format as 1/2097. In fact, most tracks are mirrored versions of 2097 with new textures and models. I know somebody on Wipeout forums (aybe) already dumped the tracks from the rom.

Being able to just play those new tracks using W1 rewrite (keeping everything else intact) would already be great.

**Comment by alexmyczko** [comment]
and what about wip3out?

**Comment by phoboslab** [comment]
Yeah, tracks/ships from 2097 should be possible, but there's no source available. Physics, HUD, menus and weapon behavior would need reverse engineering, which I don't know much about and can't justify the time needed – 2097 is my least favorite of the PSX trilogy...

wip3out has a completely different engine and data formats. If anyone would "find" the source for it, I'd certainly port it. But otherwise: nope.

**Comment by panmarco83** [comment]
Well, sorry to say that 2097 was and still is my all-time favorite.
The first was too unforgiving when slightly brushing the walls and that was the main reason I really liked 2097 so much more than the first.
If the original had the same allowance for brushing walls, I'd probably find it a lot more playable than I currently do.

I'm wondering if it could be simulated by way of acceleration direction and collision normals of the track and the ships.
This might add a bit too much for Amiga and other low powered systems, so it might then need to be a branch/fork or something :thinking:

**Comment by tigrouind** [comment]
> The first was too unforgiving when slightly brushing the walls and that was the main reason I really liked 2097 so much more than the first. If the original had the same allowance for brushing walls, I'd probably find it a lot more playable than I currently do.
> 
> I'm wondering if it could be simulated by way of acceleration direction and collision normals of the track and the ships. 
> Possibly an option in the settings menu or a flag at compile time ?

This has already been discussed [here](https://github.com/phoboslab/wipeout-rewrite/issues/13). Somebody gives a patch.

**Comment by tigrouind** [comment]
If Wipeout 1 tracks get replaced by Wipeout 2097 one, it works out of the box (except for a few tracks, see [here](https://github.com/wipeout-phantom-edition/wipeout-phantom-edition/issues/5#issuecomment-2379704329) for more info). It's missing many things like menus, scene animated models, new physics and so on but it's playable.

https://github.com/user-attachments/assets/ac06a57a-abfb-4e99-baf9-df0e49aec56c
